scripts/main/settings.js

```diff
@@ -19,10 +19,14 @@ settings.createLogin = function () {
 	 * @returns {boolean}
 	 */
 	const errorHandler = function (jqXHR, params, lycheeException) {
-		let htmlBody = "<p>" + lychee.locale["ERROR_LOGIN"] + "</p>";
-		htmlBody += lycheeException ? "<p>" + lycheeException.message + "</p>" : "";
 		basicModal.show({
-			body: htmlBody,
+			body: "<p></p><p></p>",
+			readyCB: (formElement, dialog) => {
+				/** @type {NodeList<HTMLParagraphElement>} */
+				const paragraphs = dialog.querySelectorAll("p");
+				paragraphs.item(0).textContent = lychee.locale["ERROR_LOGIN"];
+				paragraphs.item(1).textContent = lycheeException ? "<p>" + lycheeException.message + "</p>" : "";
```

Comment:
```suggestion
				paragraphs.item(1).textContent = lycheeException ? lycheeException.message : "";
```

`body` already contains a second set of `<p>`/`</p>`, doesn't it?

Reply:
Fixed in https://github.com/LycheeOrg/Lychee-front/commit/b60fdedf62f76fa77c29ff7dda2e3ce08e9d79e6.

Your are totally right. It is even harmful, because `<p>` would be shown literally. This is how `textContent` works.